packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -450,23 +462,22 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
             const container = await loadContainer(summaryVersion);
             await sendOpToUpdateSummaryTimestampToNow(container);
 
-            // Requesting a tombstoned datastore should fail!
-            await assert.rejects(async () => requestFluidObject<ITestDataObject>(container, unreferencedId),
-                (error) => {
-                    const correctErrorType = error.code === 404;
-                    const correctErrorMessage = error.message === `Datastore removed by gc: ${unreferencedId}`;
-                    return correctErrorType && correctErrorMessage;
-                },
-                `Should not be able to retrieve a tombstoned datastore in non-summarizer clients`,
-            );
-            await assert.doesNotReject(
-                async () => requestFluidObject<ITestDataObject>(summarizingContainer, unreferencedId),
-                `Should be able to retrieve a tombstoned datastore in summarizer clients`,
-            );
+            const tombstoneErrorResponse = await containerRuntime_resolveHandle(container, { url: unreferencedId });
+            assert.equal(tombstoneErrorResponse.status, 404, "Should not be able to retrieve a tombstoned datastore in non-summarizer clients");
+            assert.equal(tombstoneErrorResponse.value, `Datastore removed by gc: ${unreferencedId}`, "Expected the Tombstone error message");
+            assert.equal(tombstoneErrorResponse.headers?.[TombstoneResponseHeaderKey], true, "Expected the Tombstone header");
+
+            const tombstoneSuccessResponse = await containerRuntime_resolveHandle(container, { url: unreferencedId, headers: { [AllowTombstoneRequestHeaderKey]: true }});
+            assert.equal(tombstoneSuccessResponse.status, 200, "Should be able to retrieve a tombstoned datastore given the allowTombstone header");
+            assert.notEqual(tombstoneSuccessResponse.headers?.[TombstoneResponseHeaderKey], true, "DID NOT Expect tombstone header to be set on the response");
+
+            const summarizerResponse = await containerRuntime_resolveHandle(summarizingContainer, { url: unreferencedId });
+            assert.equal(summarizerResponse.status, 200, "Should be able to retrieve a tombstoned datastore in summarizer clients");
+            assert.notEqual(summarizerResponse.headers?.[TombstoneResponseHeaderKey], true, "DID NOT Expect tombstone header to be set on the response");
         });
 
-        // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
-        itExpects("Requesting tombstoned datastores fails in summarizing container loaded before sweep timeout",
+        // SKIPPED: This test is redundant with the previous one; SweepReady calculation/timers has coverage elsewhere
```

Comment:
Can we just delete these tests? Also, can we just delete these tests in a separate PR? This doesn't seem like a necessary change we need to backport